src/app/(sidebar)/endpoints/[[...pages]]/page.tsx

```diff
@@ -693,7 +706,7 @@ export default function Endpoints() {
 
                 const error = component.validate?.(value, isRequired);
 
-                if (error) {
+                if (error && error.result !== "success") {
```

Comment:
I added this conditional so that successful message won't be included in `FormError` (successful tx.status was included)